rcl/test/rcl/test_get_node_names.cpp

```diff
@@ -81,7 +83,12 @@ TEST_F(CLASSNAME(TestGetNodeNames, RMW_IMPLEMENTATION), test_rcl_get_node_names)
   ret = rcl_get_node_names(node1_ptr, node1_options.allocator, &node_names);
   ASSERT_EQ(RCUTILS_RET_OK, ret) << rcl_get_error_string_safe();
 
-  EXPECT_EQ(size_t(2), node_names.size);
+  std::stringstream ss;
+  ss << "Found node names:" << std::endl;
```

Comment:
I think that tests are always ran sequentially. But it's always good to have a more specific error message!